src/infra/workload/globalresources/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.94.0"
+      version = "3.97.1"
```

Comment:
how about updating TF everywhere to 3.103.1? otherwise we'll have to update right again afterwards :)

Reply:
Still on it. You might have seen that I've already updated some.

Reply:
But I'm not planning to incorporate any of the other dependabot updates here. The first e2e run already went though - need to change it once again with the changed TF params now. https://dev.azure.com/mission-critical/Online/_build/results?buildId=3360

Reply:
I assume there was a reason you went to 3.101.0 instead of latest 3.103.1 ;)